src/effects.js

```diff
@@ -33,14 +33,30 @@ export function take(pattern) {
     };
 }
 
+function getErrorObject(value) {
+    if (value && value.stack) {
+        return {
+            message: 'Unhandled exception in tale: ' + value,
+            stack: value.stack,
+        };
+    }
+
+    if (typeof value === 'object') {
+        return {
+            message: 'Unhandled exception in tale: ' + JSON.stringify(value, null, '\t'),

```

Comment:
not sure we need it pretty printed - does it help for kibana?

Reply:
Fair point, it probably just adds noise. We don't do it for other unhandled errors.